kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/config/KmsCacheConfig.java

```diff
@@ -33,14 +33,14 @@ public record KmsCacheConfig(
     }
 
     @SuppressWarnings("java:S1905") // Sonar's warning about this is incorrect, the cast is required.
-    KmsCacheConfig(Integer decryptedDekCacheSize,
-                   Long decryptedDekExpireAfterAccessSeconds,
-                   Integer resolvedAliasCacheSize,
-                   Long resolvedAliasExpireAfterWriteSeconds,
-                   Long resolvedAliasRefreshAfterWriteSeconds,
-                   Long notFoundAliasExpireAfterWriteSeconds,
-                   Long decryptedDekRefreshAfterWriteSeconds,
-                   Long decryptedDekExpireAfterWriteSeconds) {
+    public KmsCacheConfig(Integer decryptedDekCacheSize,
```

Comment:
@tombentley revert to package access.

Reply:
Turns out it had to be public, because the call site as moved to RecordEncryption, which is a different package.